demos/python_demos/llm_python_backend/README.md

```diff
@@ -0,0 +1,54 @@
+# Llama demo with python node {#ovms_demo_python_llama}
+
+## Build image
+
+From the root of the repository run:
+
+```bash
+MEDIAPIPE_DISABLE=0 PYTHON_DISABLE=0 OV_USE_BINARY=1 RUN_TESTS=0 make python_image
+```
+
+## Download model
+
+We are going to use [red-pajama-3b-chat](https://huggingface.co/togethercomputer/RedPajama-INCITE-Chat-3B-v1) model in this scenario.  
+Download the model using `download_model.py` script:
+
+```bash
+pip install -r requirements.txt
+python3 download_model.py
+```
+
+The model will appear in `./model` directory.
+
+## Run Model Server
+
+Mount the `./model` directory with the model.  
+Mount the `./servable` which contains:
+- `model.py` and `config.py` - python scripts which are required for execution and uses optimum-intel framework
+- `config.json` - which defines which servables should be loaded
+- `graph.pbtxt` - which defines MediaPipe graph containing python calculator
+
+```bash
+docker run -it --rm -p 9000:9000 -v $PWD/servable:/workspace -v $PWD/model:/model -e http_proxy=$http_proxy -e https_proxy=$https_proxy -e no_proxy=$no_proxy openvino/model_server:py --config_path /workspace/config.json --log_level DEBUG --port 9000
```

Comment:
1. are proxy envs needed? we usually don't use it in examples
2. is log_level DEBUG required?
```suggestion
docker run -it --rm -p 9000:9000 -v $PWD/servable:/workspace -v $PWD/model:/model openvino/model_server:py --config_path /workspace/config.json --port 9000
```